src/Atmosphere/CRTM_Atmosphere_Define.f90

```diff
@@ -2462,7 +2462,7 @@ FUNCTION Read_Record( &
 
     ! RH APPROACH #2
     ! Compute the relative humidity
-    CALL Compute_Relative_Humidity( atm )
+    !CALL Compute_Relative_Humidity( atm )
```

Comment:
It is okay we switch to approach 1, especially for JEDI.

JEDI does not use function `Read_Record` for input profiles, but assign all data directly from geovals. For now there is no relative humidity assigned in UFO that `atm%Relative_Humidity = 0`. This should be updated later if GOCART-GEOS5 and NAAPS tables are needed for tests later.

Reply:
@chengdang before we come back to this, I'd like to do a complete code review, cleanup.  there's lots of "comments" left in the code that need to be removed.

The other thing that needs to be done is an assessment of openMP again.   I haven't tested with multi-thread openMP yet, I'll do that now.